src/api/java/de/teamlapen/vampirism/api/event/ModEventFactory.java

```diff
@@ -0,0 +1,65 @@
+package de.teamlapen.vampirism.api.event;
+
+import de.teamlapen.vampirism.api.entity.factions.IFaction;
+import de.teamlapen.vampirism.api.entity.factions.IPlayableFaction;
+import de.teamlapen.vampirism.api.world.IVampirismVillage;
+
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.entity.passive.EntityVillager;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.world.World;
+import net.minecraft.world.gen.structure.StructureBoundingBox;
+import net.minecraftforge.common.MinecraftForge;
+import net.minecraftforge.fml.common.eventhandler.Event.Result;
+
+import java.util.List;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+public class ModEventFactory {
+
+    public static boolean fireVillagerCaptureEvent(@Nullable IVampirismVillage village, @Nonnull List<EntityVillager> villagerIn, @Nullable IPlayableFaction<?> controllingFactionIn, @Nonnull IPlayableFaction<?> capturingFactionIn) {
```

Comment:
We don't really need a dedicated ModEventFactory. Forge only does so to minimize the patches to vanilla code.
However, we can keep it, but move it into the main code package (e.g. util), no reason to have this in the api package.

Probably want to move the few other events Vampirism has to this class as well